components/atom/button/src/index.js

```diff
@@ -213,6 +218,10 @@ AtomButton.propTypes = {
    * Size of button 'small' (default), 'large'
    */
   size: PropTypes.oneOf(Object.values(SIZES)),
+  /**
+   * Align text 'center' (default), 'left'
+   */
+  alignText: PropTypes.bool,
```

Comment:
Should be a String?

Reply:
Even Better, `alignText: PropTypes.oneOf(Object.keys(ALIGN_TEXT))`

Reply:
Ups!